a10_octavia/controller/worker/tasks/a10_network_tasks.py

```diff
@@ -776,24 +776,36 @@ def execute(self, vthunder, lb_resource, vrid_list, subnet,
 
         vrid_floating_ips = []
         update_vrid_flag = False
+        existing_fips = []
         vrid_list = self._add_vrid_to_list(vrid_list, subnet, lb_resource.project_id)
         for vrid in vrid_list:
-            subnet = self.network_driver.get_subnet(vrid.subnet_id)
+            vrid_summary = self.axapi_client.vrrpa.get(vrid.vrid)
```

Comment:
can we handle exception for this axapi error?
when axapi failed, can we just use empty list in existing_fips? or we just raise the exception and do revert flow?

Reply:
Added exception handling here and set `vrid_summary = {}` and thus `existing_fips = []` if exception arises at this call.